Source/Core/DolphinQt/Config/WiimoteControllersWidget.cpp

```diff
@@ -79,13 +79,21 @@ void WiimoteControllersWidget::StartBluetoothAdapterRefresh()
 
   m_bluetooth_adapter_scan_in_progress = true;
 
+  // An artifical minimum time so "Refreshing..." is readable even if a scan completes instantly.
```

Comment:
[Super nit] If we're quoting it, the text in the comment should match the combo text to avoid confusing anyone reading this later.